client/python/polaris/management/models/__init__.py

```diff
@@ -33,15 +34,18 @@
 
 # import models into model package
 from polaris.management.models.add_grant_request import AddGrantRequest
+from polaris.management.models.authentication_parameters import AuthenticationParameters
```

Comment:
@XJDKC it seems like this is a broken reference; I can't find the namespace `polaris.management.models.authentication_parameters`. Do I need to update the script to ensure these new types get generated?

Reply:
example failure:
https://github.com/apache/polaris/actions/runs/14726054328/job/41328843247?pr=1347#step:6:3518

Reply:
Dennis and I haven't got a chance to generate the connection config related new models.